stable/prometheus/README.md

```diff
@@ -167,6 +167,7 @@ Parameter | Description | Default
 `nodeExporter.extraConfigmapMounts` | Additional node-exporter configMap mounts | `[]`
 `nodeExporter.nodeSelector` | node labels for node-exporter pod assignment | `{}`
 `nodeExporter.podAnnotations` | annotations to be added to node-exporter pods | `{}`
+`nodeExporter.pod.labels` | labels to be added to node-exporter pods | `{}`
```

Comment:
To be consistent with `podAnnotations`, call it `podLabels`.

Reply:
What about `kubeStateMetrics.pod.labels` in https://github.com/kubernetes/charts/blob/master/stable/prometheus/templates/kube-state-metrics-deployment.yaml#L24#L26 ? Should we consider to change it as well?

Reply:
@unguiculus do we have a document explaining what's the right nomenclature for labels and annotations? I don't think it's a issue to use pod.labels if they were used before by another object.

Reply:
I didn't see it was used as such before. So I guess we can go with that. It's not ideal. Maybe we can fix this inconsistency in another PR. There's no specific best practice for this, but the additional level of nesting just for one value is not ideal.